### DISCUSSION TITLE
[window-state][v2] macOS using plugins causes apps to get stuck

### DISCUSSION THREAD

**Discussion by ayangweb** [tauri-apps/tauri#11342] [Q&A]
demo: https://github.com/ayangweb/test-window-state

macOS video: 

The mouse is always loading, but the screen recorder doesn't record it.

https://github.com/user-attachments/assets/ffeeac64-74e0-4186-95ea-1a3a9a52a219

Windows video:

https://github.com/user-attachments/assets/d6037443-29c0-4826-8886-4de7c838f512

[More demo videos](https://github.com/tauri-apps/plugins-workspace/issues/1916)

**Comment by FabianLars** [comment]
Duplicate of https://github.com/tauri-apps/plugins-workspace/issues/1918 (please stop 🙃)
